src/biokbase/narrative/tests/test_job.py

```diff
@@ -567,12 +547,9 @@ def test_job_finished(self):
         }
 
         for job_id in is_finished.keys():
-            job = create_job_from_ee2(job_id)
+            job = create_job_from_ee2(job_id, mode="attributes")
             self.assertEqual(job.is_finished(), is_finished[job_id])
 
-    @mock.patch(
-        "biokbase.narrative.jobs.specmanager.SpecManager.get_spec", get_app_data
-    )
```

Comment:
Forced loading of mock app specs into `SpecManager` in `setUpClass`, so don't need to mock individual tests